07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<p>I'm not following this. You create a directory that I assume is to save the dataframe to, but you don't use the same variable names. Does <code>cfg.final_coefs_df</code> points to a filepath? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I usually like to print the full output path where I'm saving the dataframe, just to have that info here and quickly know where things are being saved to (otherwise I need to go the config to remember the output filepath).</p>